nixos/tests/libinput.nix

```diff
@@ -0,0 +1,36 @@
+import ./make-test-python.nix ({ ... }:
+
+{
+  name = "libinput";
+
+  machine = { ... }:
+    {
+      imports = [
+        ./common/x11.nix
+        ./common/user-account.nix
+      ];
+
+      test-support.displayManager.auto.user = "alice";
+
+      services.xserver.libinput = {
+        enable = true;
+        mouse = {
+          accelProfile = "flat";
+          accelSpeed = "-0.5";
+        };
+      };
+    };
+
+  testScript = ''
+    def expect_xserver_option(option, value):
+        machine.succeed(f"""cat /var/log/X.0.log | grep -F 'Option "{option}" "{value}"'""")
```

Comment:
Can we cat the X.0.log when this fails? Maybe that helps us find the root cause of the test failure.

Reply:
Done.

Reply:
Looking at the logs, it seems that `aarch64-linux` reports completely different names (like `AccelerationScheme` instead of `AccelProfile`).

I think I will change the tests to avoid those more problematic options.